lib/mnesia/src/mnesia.erl

```diff
@@ -363,28 +404,60 @@ non_transaction(State, Fun, Args, ActivityKind, Mod)
 non_transaction(_State, Fun, Args, _ActivityKind, _Mod) ->
     {aborted, {badarg, Fun, Args}}.
 
+-spec async_dirty(Fun) -> Res | no_return() when
```

Comment:
I know it have no meaning for dialyzer,  but I believe it does for documentation,
the transaction functions catch errors and return {aborted, Reason}